lib/rules/no-duplicate-selectors/__tests__/index.js

```diff
@@ -81,8 +85,10 @@ testRule({
 			warnings: [
 				{
 					message: messages.rejected('a', 1),
-					line: 1,
+					line: 2,
```

Comment:
I believe this `line` was previously incorrect; both `a` selectors are on lines `2` and `3` respectively, so I'm not sure how `1` would be reported.

If that is true, what does this mean for the test process? Is it because we didn't report a start index either?